src/psygnal/containers/_evented_list.py

```diff
@@ -58,8 +58,8 @@ class ListEvents(SignalGroup):
         `(index, value)` emitted after `value` is removed at `index`
     moving : Signal[int, int]
         `(index, new_index)` emitted before an item is moved from `index` to `new_index`
-    moved : Signal[int, int]
-        `(index, new_index, value)` emitted after `value` is moved from `index` to
+    moved : Signal[tuple, int]
```

Comment:
actually it looks like it should be `Signal[int, int, int]` ... see line 283, where the actual emit happens